# glib-networking Italian translation
# Copyright (C) 2011, 2012, 2013, 2016, 2017, 2018, 2019, 2020, 2021 Free Software Foundation, Inc
# This file is distributed under the same license as the glib-networking package.
# Luca Ferretti <lferrett@gnome.org>, 2011, 2012.
# Milo Casagrande <milo@ubuntu.com>, 2013, 2017, 2018, 2019, 2020, 2021.
# Gianvito Cavasoli <gianvito@gmx.it>, 2016.
#
msgid ""
msgstr ""
"Project-Id-Version: glib-networking\n"
"Report-Msgid-Bugs-To: https://gitlab.gnome.org/GNOME/glib-networking/issues\n"
"POT-Creation-Date: 2021-06-25 16:32+0000\n"
"PO-Revision-Date: 2021-08-18 13:23+0200\n"
"Last-Translator: Milo Casagrande <milo@milo.name>\n"
"Language-Team: Italiano <gnome-it-list@gnome.org>\n"
"Language: it\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Generator: Poedit 2.4.2\n"

#: proxy/libproxy/glibproxyresolver.c:164
msgid "Proxy resolver internal error."
msgstr "Errore interno del resolver proxy."

#: tls/base/gtlsconnection-base.c:562 tls/base/gtlsinputstream.c:78
#: tls/base/gtlsinputstream.c:141 tls/base/gtlsoutputstream.c:78
#: tls/base/gtlsoutputstream.c:143
msgid "Connection is closed"
msgstr "La connessione è chiusa"

#. Cannot perform a blocking operation during a handshake on the
#. * same thread that triggered the handshake. The only way this can
#. * occur is if the application is doing something weird in its
#. * accept-certificate callback. Allowing a blocking op would stall
#. * the handshake (forever, if there's no timeout). Even a close
#. * op would deadlock here.
#.
#: tls/base/gtlsconnection-base.c:636
msgid "Cannot perform blocking operation during TLS handshake"
msgstr "Impossibile eseguire un'operazione bloccante durante l'handshake TLS"

#: tls/base/gtlsconnection-base.c:701 tls/base/gtlsconnection-base.c:1249
#: tls/openssl/gtlsconnection-openssl.c:358
msgid "Socket I/O timed out"
msgstr "I/O sul socket scaduto"

#: tls/base/gtlsconnection-base.c:875
msgid "Server required TLS certificate"
msgstr "Il server richiede un certificato TLS"

#: tls/base/gtlsconnection-base.c:1451
#, c-format
msgid "Handshake is not finished, no channel binding information yet"
msgstr ""
"L'handshake non è finito, nessuna informazione di binding del canale "
"disponibile"

#: tls/base/gtlsconnection-base.c:1530
msgid "Peer does not support safe renegotiation"
msgstr "Il nodo non supporto la rinegoziazione sicura"

#: tls/base/gtlsconnection-base.c:1678 tls/gnutls/gtlsconnection-gnutls.c:482
#: tls/openssl/gtlsconnection-openssl.c:209
#: tls/openssl/gtlsconnection-openssl.c:834
#, c-format
msgid "Unacceptable TLS certificate"
msgstr "Certificato TLS inammissibile"

#: tls/base/gtlsconnection-base.c:2152
#, c-format
msgid "Receive flags are not supported"
msgstr "I flag di ricezione non sono supportati"

#: tls/base/gtlsconnection-base.c:2299
#, c-format
msgid "Send flags are not supported"
msgstr "I flag di invio non sono supportati"

#: tls/gnutls/gtlscertificate-gnutls.c:361
#: tls/openssl/gtlscertificate-openssl.c:362
#, c-format
msgid "Could not parse DER certificate: %s"
msgstr "Impossibile analizzare il certificato DER: %s"

#: tls/gnutls/gtlscertificate-gnutls.c:382
#: tls/openssl/gtlscertificate-openssl.c:382
#, c-format
msgid "Could not parse PEM certificate: %s"
msgstr "Impossibile analizzare il certificato PEM: %s"

#: tls/gnutls/gtlscertificate-gnutls.c:405
#: tls/openssl/gtlscertificate-openssl.c:401
#, c-format
msgid "Could not parse DER private key: %s"
msgstr "Impossibile analizzare la chiave privata DER: %s"

#: tls/gnutls/gtlscertificate-gnutls.c:428
#: tls/openssl/gtlscertificate-openssl.c:420
#, c-format
msgid "Could not parse PEM private key: %s"
msgstr "Impossibile analizzare la chiave privata PEM: %s"

#: tls/gnutls/gtlscertificate-gnutls.c:455
#, c-format
msgid "Could not import PKCS #11 certificate URI: %s"
msgstr "Impossibile importare l'URI del certificato PKCS #11: %s"

#: tls/gnutls/gtlscertificate-gnutls.c:497
#: tls/openssl/gtlscertificate-openssl.c:455
msgid "No certificate data provided"
msgstr "Nessun dato di certificato fornito"

#: tls/gnutls/gtlscertificate-gnutls.c:806
#: tls/gnutls/gtlsconnection-gnutls.c:1033
#, c-format
msgid "Cannot verify peer identity of unexpected type %s"
msgstr "Impossibile verificare l'identità peer di tipo imprevisto %s"

#: tls/gnutls/gtlsconnection-gnutls.c:198
msgid "Could not create TLS connection:"
msgstr "Impossibile creare la connessione TLS:"

#: tls/gnutls/gtlsconnection-gnutls.c:208
#: tls/gnutls/gtlsconnection-gnutls.c:228
#: tls/openssl/gtlsclientconnection-openssl.c:458
#: tls/openssl/gtlsserverconnection-openssl.c:503
#, c-format
msgid "Could not create TLS connection: %s"
msgstr "Impossibile creare la connessione TLS: %s"

#: tls/gnutls/gtlsconnection-gnutls.c:426
#: tls/gnutls/gtlsconnection-gnutls.c:437
#: tls/gnutls/gtlsconnection-gnutls.c:451
#: tls/openssl/gtlsconnection-openssl.c:176
#, c-format
msgid "Peer failed to perform TLS handshake: %s"
msgstr "Il nodo non è stato in grado di eseguire l'handshake TLS: %s"

#: tls/gnutls/gtlsconnection-gnutls.c:459
msgid "TLS connection closed unexpectedly"
msgstr "La connessione TLS si è chiusa in modo inatteso"

#: tls/gnutls/gtlsconnection-gnutls.c:474
#: tls/openssl/gtlsconnection-openssl.c:201
msgid "TLS connection peer did not send a certificate"
msgstr "Il nodo di connessione TLS non ha inviato un certificato"

#: tls/gnutls/gtlsconnection-gnutls.c:490
#, c-format
msgid "Peer sent fatal TLS alert: %s"
msgstr "Il nodo ha inviato un avviso TLS fatale: %s"

#: tls/gnutls/gtlsconnection-gnutls.c:500
msgid "Protocol version downgrade attack detected"
msgstr "Rilevato attacco di declassamento del protocollo di versione"

#: tls/gnutls/gtlsconnection-gnutls.c:509
#, c-format
msgid "Message is too large for DTLS connection; maximum is %u byte"
msgid_plural "Message is too large for DTLS connection; maximum is %u bytes"
msgstr[0] ""
"Il messaggio è troppo grande per una connessione DTLS (massimo %u byte)"
msgstr[1] ""
"Il messaggio è troppo grande per una connessione DTLS (massimo %u byte)"

#: tls/gnutls/gtlsconnection-gnutls.c:518
msgid "The operation timed out"
msgstr "Tempo esaurito per l'operazione"

#: tls/gnutls/gtlsconnection-gnutls.c:870
#, c-format
msgid "Error performing TLS handshake: %s"
msgstr "Errore nell'eseguire l'handshake TLS: %s"

#: tls/gnutls/gtlsconnection-gnutls.c:973
#: tls/openssl/gtlsconnection-openssl.c:618
msgid "Error performing TLS handshake"
msgstr "Errore nell'eseguire l'handshake TLS"

#: tls/gnutls/gtlsconnection-gnutls.c:1152
#, c-format
msgid "Channel binding type is not implemented in the TLS library"
msgstr ""
"Il tipo di associazione del canale non è implementato nella libreria TLS"

#: tls/gnutls/gtlsconnection-gnutls.c:1156
#, c-format
msgid "Channel binding data is not yet available"
msgstr "I dati di associazione del canale non sono ancora disponibili"

#: tls/gnutls/gtlsconnection-gnutls.c:1194
#: tls/gnutls/gtlsconnection-gnutls.c:1206
#, c-format
msgid "X.509 certificate is not available on the connection"
msgstr "Il certificato X.509 non è disponibile sulla connessione"

#: tls/gnutls/gtlsconnection-gnutls.c:1219
#, c-format
msgid "X.509 certificate is not available or is of unknown format: %s"
msgstr "Il certificato X.509 non è disponibile o è di formato sconosciuto: %s"

#: tls/gnutls/gtlsconnection-gnutls.c:1230
#: tls/openssl/gtlsconnection-openssl.c:709
#, c-format
msgid "Unable to obtain certificate signature algorithm"
msgstr "Impossibile ottenere l'algoritmo di firma del certificato"

#: tls/gnutls/gtlsconnection-gnutls.c:1246
#: tls/openssl/gtlsconnection-openssl.c:729
#, c-format
msgid ""
"Current X.509 certificate uses unknown or unsupported signature algorithm"
msgstr ""
"Il certificato X.509 attuale utilizza un algoritmo di firma sconosciuto o "
"non supportato"

#: tls/gnutls/gtlsconnection-gnutls.c:1340
#: tls/openssl/gtlsconnection-openssl.c:809
#, c-format
msgid "Requested channel binding type is not implemented"
msgstr "Il tipo di associazione del canale richiesto non è implementato"

#: tls/gnutls/gtlsconnection-gnutls.c:1361
#: tls/gnutls/gtlsconnection-gnutls.c:1421
#: tls/openssl/gtlsconnection-openssl.c:827
#: tls/openssl/gtlsconnection-openssl.c:923
msgid "Error reading data from TLS socket"
msgstr "Errore nel leggere dati dal socket TLS"

#. flags
#: tls/gnutls/gtlsconnection-gnutls.c:1443
#: tls/gnutls/gtlsconnection-gnutls.c:1506
#: tls/openssl/gtlsconnection-openssl.c:1002
msgid "Error writing data to TLS socket"
msgstr "Errore nello scrivere dati sul socket TLS"

#: tls/gnutls/gtlsconnection-gnutls.c:1476
#, c-format
msgid "Message of size %lu byte is too large for DTLS connection"
msgid_plural "Message of size %lu bytes is too large for DTLS connection"
msgstr[0] "Un messaggio di %lu byte è troppo grande per la connessione DTLS"
msgstr[1] "Un messaggio di %lu byte è troppo grande per la connessione DTLS"

#: tls/gnutls/gtlsconnection-gnutls.c:1478
#, c-format
msgid "(maximum is %u byte)"
msgid_plural "(maximum is %u bytes)"
msgstr[0] "(massimo %u byte)"
msgstr[1] "(massimo %u byte)"

#: tls/gnutls/gtlsconnection-gnutls.c:1525
#, c-format
msgid "Error performing TLS close: %s"
msgstr "Errore nell'eseguire la chiusura TLS: %s"

#: tls/gnutls/gtlsdatabase-gnutls.c:688
msgid ""
"Failed to load system trust store: GnuTLS was not configured with a system "
"trust"
msgstr ""
"Caricamento dell'archivio di attendibilità di sistema non riuscito: GnuTLS "
"non è stato configurato con un archivio di attendibilità di sistema"

#: tls/gnutls/gtlsdatabase-gnutls.c:693 tls/openssl/gtlsdatabase-openssl.c:229
#, c-format
msgid "Failed to load system trust store: %s"
msgstr "Caricamento dell'archivio di attendibilità di sistema non riuscito: %s"

#: tls/gnutls/gtlsfiledatabase-gnutls.c:153
#: tls/openssl/gtlsfiledatabase-openssl.c:454
#, c-format
msgid "Failed to populate trust list from %s: %s"
msgstr "Caricamento dell'elenco di attendibilità da %s non riuscito: %s"

#: tls/gnutls/gtlsserverconnection-gnutls.c:124
#: tls/openssl/gtlsserverconnection-openssl.c:172
#: tls/openssl/gtlsserverconnection-openssl.c:228
msgid "Certificate has no private key"
msgstr "Il certificato non presenta chiave privata"

#: tls/openssl/gtlsclientconnection-openssl.c:317
#: tls/openssl/gtlsserverconnection-openssl.c:346
#, c-format
msgid "Could not set TLS cipher list: %s"
msgstr "Impossibile impostare l'elenco dei cifrari TLS: %s"

#: tls/openssl/gtlsclientconnection-openssl.c:343
#: tls/openssl/gtlsserverconnection-openssl.c:372
#, c-format
msgid "Could not set MAX protocol to %d: %s"
msgstr "Impossibile impostare il protocollo MAX su %d: %s"

#: tls/openssl/gtlsclientconnection-openssl.c:404
#: tls/openssl/gtlsserverconnection-openssl.c:433
#, c-format
msgid "Could not create TLS context: %s"
msgstr "Impossibile creare il contesto TLS: %s"

#: tls/openssl/gtlsconnection-openssl.c:217
#, c-format
msgid "Unacceptable TLS certificate authority"
msgstr "Autorità certificato TLS inammissibile"

#: tls/openssl/gtlsconnection-openssl.c:225
msgid "Digest too big for RSA key"
msgstr "Digest troppo grande per una chiave RSA"

#: tls/openssl/gtlsconnection-openssl.c:234
msgid "Secure renegotiation is disabled"
msgstr "Rinegoziazione sicura disabilitata"

#: tls/openssl/gtlsconnection-openssl.c:253
#, c-format
msgid "%s: The connection is broken"
msgstr "%s: la connessione è rotta"

#: tls/openssl/gtlsconnection-openssl.c:678
#, c-format
msgid "Channel binding data tls-unique is not available"
msgstr "I dati di associazione del canale tls-unique non sono disponibili"

#: tls/openssl/gtlsconnection-openssl.c:701
#, c-format
msgid "X.509 Certificate is not available on the connection"
msgstr "Il certificato X.509 non è disponibile sulla connessione"

#: tls/openssl/gtlsconnection-openssl.c:747
#, c-format
msgid "Failed to generate X.509 certificate digest"
msgstr "Impossibile generare il digest del certificato X.509"

#: tls/openssl/gtlsconnection-openssl.c:778
#, c-format
msgid "TLS Connection does not support TLS-Exporter feature"
msgstr "La connessione TLS non supporta la funzionalità TLS-Exporter"

#: tls/openssl/gtlsconnection-openssl.c:781
#, c-format
msgid "Unexpected error while exporting keying data"
msgstr "Errore imprevisto nell'esportare i dati di keying"

#: tls/openssl/gtlsconnection-openssl.c:1063
msgid "Error performing TLS close"
msgstr "Errore nell'eseguire la chiusura TLS"

#: tls/openssl/gtlsdatabase-openssl.c:197
msgid "Could not get trusted anchors from Keychain"
msgstr "Impossibile ottenere riferimenti attendibili dal portachiavi"

#: tls/openssl/gtlsdatabase-openssl.c:269
msgid "Could not create CA store"
msgstr "Impossibile creare l'archivio CA"

#: tls/openssl/gtlsserverconnection-openssl.c:179
#: tls/openssl/gtlsserverconnection-openssl.c:247
#, c-format
msgid "There is a problem with the certificate private key: %s"
msgstr "C'è un problema con la chiave privata del certificato: %s"

#: tls/openssl/gtlsserverconnection-openssl.c:188
#: tls/openssl/gtlsserverconnection-openssl.c:239
#, c-format
msgid "There is a problem with the certificate: %s"
msgstr "C'è un problema con il certificato: %s"
